documentation/plain_rules.md

```diff
@@ -76,7 +76,15 @@ The `type` field is mandatory and must be one of
 * `post`: make an HTTP POST
 * `twitter`: send a twitter
 
-The actions `sms`, `email`, `post` include a field `template` used to build the body of message/request. This text can include placeholders for attributes of the generated event. That placeholder has the form `${` *attribute* `}` which will be substituted by the value of the event's attribute.  The name of the attribute must be prefixed with "ev__" if no alias in the `select` clause is provided. All alias for simple event attributes or "complex" calculated values can be directly used in the placeholder with their name. Any of the original event attributes can be referred as **ev__** *name* (double underscore), if an alias hasn't been given to them.
+
+### Attribute substitution
```

Comment:
Actually, it more than attributes, as ${id} and ${type} refer to the triggering entity. Thus, probably it would better to use:

"String substitution syntax".